content/en/blog/2022/how-to-get-started-with-opentelemetry-go.md

```diff
@@ -0,0 +1,435 @@
+---
+title: Instrument your Go web application with openTelemetry
+linkTitle: Get Started with OpenTelemetry & Go
+date: 2022-06-27
+author: Team Aspecto
+canonical_url: https://www.aspecto.io/blog/opentelemetry-go-getting-started/
+---
+
+# How to Get Started with OpenTelemetry Go
+
+In this guide, you will learn hands-on how to create and visualize traces with OpenTelemetry Go without prior knowledge.
+
+We will start with creating a simple to-do app that uses Mongo and the Gin framework. Then, we will send tracing data to Jaeger Tracing and to Aspecto for visualization. You can find all the relevant files in this [Github repository](https://github.com/aspecto-io/opentelemetry-examples/tree/master/go).
+
+!["OpenTelemetry Go The Mandalorian"](https://www.aspecto.io/wp-content/uploads/2022/06/OpenTelemetry-Go-The-Mandalorian-2048x1406.png)
+
+## What to Expect
+
+- Intro to OpenTelemetry
+- Hello world: OpenTelemetry GO example
+  - Create main.go file with Gin and Mongo
+  - Install OpenTelemetry GO
+  - Gin instrumentation: gin.Context
+- Visualization with Jaeger
+  - OpenTelemetry Go and Jaeger Tracing
+
+## Intro to OpenTelemetry
```

Comment:
This section is not required, please point to the existing concept docs that give you that description.

Reply:
@NavehMevorach you unresolved this conversation, but I don't see it resolved. what has changed?

Reply:
I'm not sure what you want me to remove. The title itself?

Reply:
the whole section, everything from line 24 to 53: this is a blog on opentelemetry.io, so either people know about OpenTelemetry already or they should read the [Concepts](https://opentelemetry.io/docs/concepts/) to learn more